keyboards/melgeek/z70ultra_v1/info.json

```
@@ -0,0 +1,388 @@
+{
+    "keyboard_name": "MelGeek Z70Ultra v1",
+    "manufacturer": "MelGeek",
+    "url": "",
+    "maintainer": "melgeek001365",
+    "usb": {
+        "vid": "0xEDED",
+        "pid": "0x6575",
+        "device_version": "0.0.1"
+    },
+    "processor": "STM32F303",
+    "bootloader": "stm32-dfu",
```

fixes CI Build failure
```suggestion
    "bootloader": "stm32-dfu",
    "board": "QMK_PROTON_C",
```

---

This isn't the proper way to solve the issue.   Unless the board is using a proton C, it shouldn't use the proton c board files.  Instead, halconf. and mcuconf.h should be provided to enable the needed hardware peripharals.

---

Thank you for your suggestion, but how can I modify it? Are there any examples for reference?

---

Yeah, np.  Mostly, it's figure out what you need, and enable that.  The moonlander was initially added using the board files, and you can see it migrated completely off of it, in this commit: 
https://github.com/qmk/qmk_firmware/commit/40578621a76b43481116fd73b10b8cdec3d3a7ef

basically, you need the halconf.h and mcuconf.h files to configure things.  And you should only enable what you need to.